packages/frontend/src/components/composer/Composer.tsx

```diff
@@ -982,7 +982,11 @@ export function useDraft(
     if (chatId == undefined || !canSend) {
       return
     }
-    const quoteMessage = (messageId: number) => {
+    const quoteMessage = async (messageId: number) => {
+      const message = await BackendRemote.rpc.getMessage(accountId, messageId)
+      if (message.isInfo || message.viewType === 'VideochatInvitation') {
+        return
+      }
```

Comment:
```suggestion
    const quoteMessage = async (messageId: number) => {
      const message = await BackendRemote.rpc.getMessage(accountId, messageId)
      if (message.isInfo || message.viewType === 'VideochatInvitation') {
        return
      }
```

This makes it impossible to "step over" an info message by pressing Ctrl + Up. It's `quoteMessage`, not `maybeQuoteMessage`.